dagshub/data_engine/model/metadata_field_builder.py

```diff
@@ -80,6 +80,57 @@ def set_annotation(self, is_annotation: bool = True) -> "MetadataFieldBuilder":
         self._set_or_unset(ReservedTags.ANNOTATION.value, is_annotation)
         return self
 
+    def set_thumbnail(self, thumbnail_type: Union[ThumbnailType, None] = None,
+                      is_thumbnail: bool = True) -> "MetadataFieldBuilder":
```

Comment:
What happens when I call it like this:
```python
set_thumbnail("image")
```
?

Honestly I think having to import another enum for this is a bit unwieldy for user side.
You can probably allow strings in thumbnail_type and then in the beginning of the function (after the is None check) do:
```python
if isinstance(thumbnail_type, str):
    thumbnail_type = ThumbnailType(thumbnail_type)
```